### PR TITLE
Fixed setting view to have updated UI and show users in groups

### PR DESCRIPTION
**Pull request by kansairob**
Changed layout of the login page and colors. Also worked on the functionality for the registration page.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,2 +1,587 @@
 {
+  "requires": true,
```
Comment (Kalatco) [inline package-lock.json:2]: this file doesn't belong to our project, remove it

File: smallTalk/settings.py [hunk 2]
```diff
@@ -25,7 +25,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = ['0.0.0.0', '192.168.0.107', '172.30.41.76']
+ALLOWED_HOSTS = ['0.0.0.0', '192.168.0.107', '172.30.41.76', '192.168.1.48']
```
Comment (Kalatco) [inline smallTalk/settings.py:28]: change to ALLOWED_HOSTS = ['*']

File: src/views/LoginView.js [hunk 3]
```diff
@@ -13,7 +13,14 @@ function SigninView(props) {
   const [emailText, setEmailText] = useState("");
   const [passwordText, setPasswordText] = useState("");
   const [isInvalidLogin, setisInvalidLogin] = useState(false);
-
+  const [isLoginForm, setIsLoginForm] = useState(true);
```
Comment (Kalatco) [inline src/views/LoginView.js:16]: the purpose of git version control is to separate different changes (or tasks) of code into separate branches. Your task to "update settings page" has nothing to do with the login form, please remove these changes and move them to a new branch

File: src/views/SettingsView.js [hunk 4]
```diff
@@ -13,10 +15,6 @@ function SettingsView(props) {
   const [currentPasswordText, setCurrentPasswordText] = useState("");
   const [newGroupText, setNewGroupText] = useState("");
 
-  const handleUserNameInput = (enteredText) => {
-    setUserNameText(enteredText);
-  }
-
   const handleFirstNameInput = (enteredText) => {
```
Comment (Kalatco) [inline src/views/SettingsView.js:18]: we talked a few days ago about removing these functions and directly calling the useState setter method for all these cases. please fix

File: smallTalk/settings.py (hunk 2 above)
Comment (Kalatco) [reply smallTalk/settings.py:28]: this will allow any host computer to serve data from the server

File: package-lock.json (hunk 1 above)
Comment (Kalatco) [reply package-lock.json:2]: but do not remove the /src/package-lock.json file

**Review by Kalatco — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (kansairob) [reply package-lock.json:2]: Updated file.

File: smallTalk/settings.py (hunk 2 above)
Comment (kansairob) [reply smallTalk/settings.py:28]: Updated file.

File: src/views/LoginView.js (hunk 3 above)
Comment (kansairob) [reply src/views/LoginView.js]: Updated file.

File: src/views/SettingsView.js (hunk 4 above)
Comment (kansairob) [reply src/views/SettingsView.js:18]: Updated file.

File: src/store/index.js [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import { createStore } from "redux";
 
 // IPv4 server address goes here
-const SERVER_ADDRESS = '192.168.0.107';
+const SERVER_ADDRESS = '192.168.1.48';
```
Comment (Kalatco) [inline src/store/index.js:4]: theres a merge conflict, change the IP to '192.168.0.107:8080'

File: src/views/LoginView.js [hunk 6]
```diff
@@ -13,7 +13,14 @@ function SigninView(props) {
...
+  const [emailReg, setEmailReg] = usestate("");
```
Comment (Kalatco) [inline src/views/LoginView.js:17]: the login view registration code is here again???

File: src/views/SettingsView.js (hunk 4 above)
Comment (Kalatco) [reply src/views/SettingsView.js:18]: issue still exists

**Review by Kalatco — CHANGES_REQUESTED** [review]

File: src/store/index.js (hunk 5 above)
Comment (kansairob) [reply src/store/index.js:4]: Fixed

File: src/views/LoginView.js (hunk 6 above)
Comment (kansairob) [reply src/views/LoginView.js:17]: Fixed

**Pull request merged**